app/views/pages/release-notes/2020/APRIL/06.liquid

```diff
@@ -0,0 +1,27 @@
+---
+converter: markdown
+metadata:
+  title: Renaming files, pos-cli clean --including-schema
+  description: March 16, 2020
+  skip_contribute_button: true
+  last_edit: false
+slug: release-notes/16-MARCH-2020
+---
+
+#### {{ page.metadata.description }}
+
+<h4 class="release-note release-note__new">NEW</h4>
+
+* added support for a the new `--including-schema` parameter of data-clean command - `pos-cli data-clean <env> --including-schema` will now remove all admin resources as well, like Forms, Model Schemas etc - effectively a shortcut to get rid of anything on the instance except of configuration (like test email, constants)
```

Comment:
This is not yet merged/released

Reply:
I see the PR has been merged without the changes, I'm now making all changes. Should I delete this bullet? Thanks!

Reply:
@Slashek @pavelloz Deleted.